kahuna/public/js/image/service.js

```diff
@@ -19,10 +19,13 @@ imageService.factory('imageService', ['editsService', function(editsService) {
         );
         const resource = image.data.userMetadata.data.usageRights;
 
-        const save = newData => editsService.update(resource, newData, image);
-        const remove = () => editsService.remove(resource, image);
+        const save = newData =>
+            editsService.update(resource, newData, image).then(resource => resource.data);
 
-        return { data, resource, image, save, remove };
+        const remove = () =>
+            editsService.remove(resource, image).then((/*resoure*/) => image.data.usageRights);
```

Comment:
I've put it there so you know that it's there. I really don't like the fact that jslint make you remove unused parameters, as they are a true representation of what's there (slightly closer to å  typed system).
Does it feel noisy?